test/jest.config.js

```diff
@@ -2,6 +2,6 @@
 
 module.exports = {
   testEnvironment: 'node',
-  testMatch: ['<rootDir>/**/*.test.js'],
+  testMatch: ['<rootDir>/**/*.test.{js,jsx,ts,tsx}'],
```

Comment:
Revert this, looks like a bad find and replace. 😄

Reply:
I found out why. It's because the `react-scripts` inside the temporary test node_modules is the one from npm (without typescript support) not a linked one with the current code.